src/system_window.cpp

```diff
@@ -568,12 +572,14 @@ void SystemWindow::EndFrame()
 
 		glEnable( GL_TEXTURE_2D );
 
+		#ifndef __APPLE__
 		glBegin( GL_QUADS );
```

Comment:
It seems like MacOS doesn't support OpenGL compatibility profile, only core profile, where `glBegin`-like function do not exist anymore. Am i right?

Reply:
fixed